common/tor/tor_proxy_uri_helper.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/tor/tor_proxy_uri_helper.h"
+#include "chrome/common/channel_info.h"
+#include "components/version_info/channel.h"
+
+namespace tor {
+
+std::string GetTorProxyURI() {
+  const std::string TorProxyScheme("socks5://");
```

Comment:
this should be `constexpr char kTorProxyScheme[] = "socks5://` in an anonymous namespace